### ISSUE TITLE
GO package naming

### ISSUE THREAD

**Issue by gstachni** [awesome-astra/sample-code#1]
It looks like we have a typo in the go sample names and directory

sample-code/astra-gprc/ (should be astra-grpc?)
-- AstraGPRCQuickStart.go (should be AstraGRPCQuickStart.go?)

**Comment by synedra** [comment]
Yep, fixing it.Sent from my iPhoneOn Apr 14, 2023, at 11:24 AM, Greg Stachnick ***@***.***> wrote:﻿
It looks like we have a typo in the go sample names and directory
sample-code/astra-gprc/ (should be astra-grpc?)
-- AstraGPRCQuickStart.go (should be AstraGRPCQuickStart.go?)
